GSPlayer/Classes/View/VideoPlayerView.swift

```diff
@@ -335,6 +337,9 @@ private extension VideoPlayerView {
     func observe(playerItem: AVPlayerItem?) {
         
         guard let playerItem = playerItem else {
+            playerBufferingObservation?.invalidate()
+            playerItemStatusObservation?.invalidate()
+            playerItemKeepUpObservation?.invalidate()
```

Comment:
I think we should add above all these statements in `deinit` as well? WDYT?

Reply:
yes, we can add those as well

Reply:
added to deinit as well